modules/react/breadcrumbs/lib/Breadcrumbs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+import {useBreadcrumbsModel} from './hooks/useBreadcrumbsModel';
+import {BreadcrumbsList} from './BreadcrumbsList';
+import {BreadcrumbsOverflowButton} from './BreadcrumbsOverflowButton';
+import {BreadcrumbsItem} from './BreadcrumbsItem';
+import {BreadcrumbsLink} from './BreadcrumbsLink';
+import {BreadcrumbsCurrentItem} from './BreadcrumbsCurrentItem';
+import {BreadcrumbsMenu} from './BreadcrumbsMenu';
+
+export interface BreadcrumbsProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The accessibility label for the nav element.
+   *
+   * @default "breadcrumbs"
+   */
+  'aria-label'?: string;
+  /**
+   * The contents of the Breadcrumbs. Can be `Breadcrumbs` children or any valid elements.
+   */
+  children: React.ReactNode;
+}
+
+export const Breadcrumbs = createContainer()({
+  displayName: 'Breadcrumbs',
+  modelHook: useBreadcrumbsModel,
+  subComponents: {
+    List: BreadcrumbsList,
+    Item: BreadcrumbsItem,
```

Comment:
```suggestion
    /**
     * ### Breadcrumbs Item
     * ---
     * [View Developer Docs](https://canvas.workdaydesign.com/components/navigation/breadcrumbs/#breadcrumbsitem)
     * 
     * List items in `Breadcrumb.List`.
     * By default, this item is truncated with a tooltip at `350px`,
     * but that can be customized with the `maxWidth` prop.
     * 
     * @example
     * ```tsx
     * <Breadcrumbs.Item maxWidth={200}>
     *   <Breadcrumbs.Link href="/docs">Docs</Breadcrumbs.Link>
     * </Breadcrumbs.Item>
     * ```
     */
    Item: BreadcrumbsItem,
```